miniapps/meshing/mesh-explorer.cpp

```diff
@@ -914,8 +914,13 @@ int main (int argc, char *argv[])
             cout << "Number of colors: " << attr.Max() + 1 << endl;
             for (int i = 0; i < mesh->GetNE(); i++)
             {
-               attr(i) = i; // coloring by element number
+               attr(i) = partitioning[i] = i; // coloring by element number
```

Comment:
I understand that this hotfix resolves the mentioned issue, and why it does that. However, I think it would be more beneficial in the long run to have an extra array for the explosion plot, as one might want to add features in the future which needs the partitioning info also during explosion plots.

Reply:
I'm not quite following the above comment. The user is free to specify the partitioning array when calling `Mesh::PrintWithPartitioning` and that's what will be shrinked/zoomed in GLVis. In this miniapp, for this particular option, we do make a special choice, but that's not limiting overall.

Maybe I am missing the point?

Reply:
Sorry, let me try to be have a more concrete example then. A user might load a mesh with 3 materials via the `-m` command line argument. Then the user generates a partitioning via `p`. Next the users presses `e` to inspect the elements. Finally, the user presses `P`, to check the partitioning. However, instead of the partitioning the user will now see the element number colors. This might be a bit confusing for users.

Reply:
Thanks for clarifying. I understand the issue now (this workflow is not how I typically use the miniapp)